ttnn/cpp/ttnn/operations/conv/conv_transpose2d/conv_transpose2d.cpp

```diff
@@ -102,8 +102,10 @@ Result conv_transpose2d(
     std::array<uint32_t, 2> dilation,
     uint32_t groups,
     std::optional<const ttnn::Tensor> bias_tensor,
-    std::optional<const conv2d::Conv2dConfig> conv_config_)   {
+    std::optional<const conv2d::Conv2dConfig> conv_config_,
+    std::optional<const DeviceComputeKernelConfig> compute_config_)   {
```

Comment:
I would do pass by const ref for compute_config_, not just here but in general

Reply:
I can make this change in a different PR.

Reply:
Yeah I think there is PR in flight that does this in general 
https://github.com/tenstorrent/tt-metal/pull/15187